### PR TITLE
add decOM

### PR DESCRIPTION
**Pull request by CamilaDuitama**
Request to add decOM to bioconda
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/decom/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    - decOM-aOralOut = decOM.aOralOut.__main__:main
+    - decOM-format = decOM.format.__main__:main
+    - decOM-LOO = decOM.LOO.__main__:main
+    - decOM-CV = decOM.CV.__main__:main
+    - decOM-MST = decOM.MST.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - colorama
```
Comment (bgruening) [inline recipes/decom/meta.yaml:26]: the only thing you need in the host section is pip and python.

File: recipes/decom/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  imports:
+    - decOM
+    - decOM.CV
+    - decOM.LOO
+    - decOM.MST
+    - decOM.aOralOut
+    - decOM.data
+    - decOM.format
+    - decOM.modules
+    - test
```
Comment (bgruening) [inline recipes/decom/meta.yaml:55]: is this really needed?

File: recipes/decom/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    - decOM-format --help
+    - decOM-LOO --help
+    - decOM-CV --help
+    - decOM-MST --help
+
+about:
+  home: "https://github.com/CamilaDuitama/decOM"
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (bgruening) [inline recipes/decom/meta.yaml:68]: the license file is missing

File: recipes/decom/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  name: decom
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f85e93758e6ac02acbbb48f75f1d4582a266124aed671b3f6a78690ab12ac75c
+
+build:
+  number: 0
+  noarch: generic
```
Comment (bgruening) [inline recipes/decom/meta.yaml:14]: should be noarch: python

**Review by bgruening — APPROVED** [review]

**Comment by jfy133** [comment]
@CamilaDuitama do you need any more help on this?

**Comment by jfy133** [comment]
@CamilaDuitama - I guess forgot the sha update with the version bump from 0.31->0.32 I guess?

File: recipes/decom/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    - python
+  run:
+    - colorama
+    - dask >=2021.12.0
+    - importlib_resources >=5.4.0
+    - python-kaleido
+    - numpy >=1.7
+    - pandas
+    - plotly >=5.5.0
+    - python
```
Comment (jfy133) [inline recipes/decom/meta.yaml:38]:
```suggestion
    - python
    - git
```

**Comment by CamilaDuitama** [comment]
@BiocondaBot please add label

**Review by jfy133 — APPROVED** [review]

**Pull request merged**